source/sync/configure/optimize-sync-atlas-usage.txt

```diff
@@ -132,16 +132,27 @@ advanced backend compaction only occurs for history older than the
       CREATE table1.obj1
 
    Because ``table2.object2`` was deleted, it has no impact on the end
-   state of the {+realm+}. As a result, advanced backend compaction can
+   state of the realm. As a result, advanced backend compaction can
    completely remove *both* the CREATE and the DELETE operations for
    ``table2.object2`` from history. However, any clients that synchronized
    the CREATE operation but not the DELETE operation will not be able
    to converge to a common end state.
 
-Advanced backend compaction only runs on {+app+}s that specify a
+Advanced backend compaction only runs on Apps that specify a
 client maximum offline time. Only history older than the client maximum
 offline time experiences advanced backend compaction.
 
+.. _trimming:
+
+Trimming
+~~~~~~~~
+
+**Trimming** optimizes backend sync storage even further than
+:ref:`advanced backend compaction <advanced-backend-compaction>`.
```

Comment:
I'd like to leave it because I forsee a lot of TSEs linking users directly to this section, and "advanced backend compaction" is a big ol' mouthful term that users might not search for in the sidebar.